zuul.d/project-templates.yaml

```diff
@@ -839,6 +839,23 @@
             vars:
               ansible_test_collections: true
 
+- project-template:
+    name: ansible-collections-security-splunk-es
+    check:
+      jobs:
+        - build-ansible-collection
+        - ansible-security-integration-splunk-es-python36:
+            vars:
+              ansible_test_collections: true
```

Comment:
this could be moved into the job stanza, as it only runs on collections now. No ansible/ansible stable branches

Reply:
Good idea.